meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -441,6 +441,8 @@ LMP_FLASHLAYOUT:stm32mp15-eval = "FlashLayout_emmc_stm32mp157a-ev1-optee.tsv"
 ## stm32mp15-disco
 LINUX_A7_EXAMPLES_DT:remove:stm32mp15-disco = "stm32mp157c-dk2-a7-examples"
 CUBE_M4_EXAMPLES_DT:remove:stm32mp15-disco = "stm32mp157c-dk2-m4-examples"
+LINUX_A7_EXAMPLES_DT:remove:stm32mp15-disco = "stm32mp157f-dk2-a7-examples"
+CUBE_M4_EXAMPLES_DT:remove:stm32mp15-disco = "stm32mp157f-dk2-m4-examples"
```

Comment:
You can just extend the lines above adding the new content instead.

Reply:
fixed